.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,13 +1,13 @@
 name: Bug Report
-description: 'Support Requests will be closed immediately, if you are not 100% certain this is a bug please go to our Reddit, Discord, Forums, or IRC first. Exceptions do not mean you found a bug!'
+description: 'Support Requests will be closed immediately, if you are not 100% certain this is a bug please go to our Reddit, Discord, Forums, or IRC first. Exceptions do not mean you found a bug! Sonarr v2 is end of life and not supported. Sonarr v4 is non-public and not supported.'
```

Comment:
I don't think there is any advantage to telling people v4 isn't supported here, maybe they won't open an issue that's actually in v4 that needs to be fixed, but that doesn't really help us.

Reply:
removed